### PR TITLE
feat: add anonymizeUser method to openApi and CLI

### PR DESCRIPTION
**Pull request by Johaney-s**
* This method was missing in openApi and cli

### CODE REVIEW

**Review by cuadradek — APPROVED** [review]

File: perun-cli/anonymizeUser [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	};
+}
+
+my ($userId, $batch);
+GetOptions ("help|h" => sub {
+		print help();
+		exit 0;
+	}, "batch|b"     => \$batch,
+	"userId|m=i"     => \$userId) || die help();
```
Comment (balcirakpeter) [inline perun-cli/anonymizeUser:26]: I think the short version of the `userId` parameter should be `u` and not `m`

File: perun-cli/anonymizeUser [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+use Getopt::Long qw(:config no_ignore_case);
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+
+sub help {
+	return qq{
+	Anonymizes a user. User id is required.
+	--------------------------------------
+	Available options:
+	--userId   | -u user id
```
Comment (balcirakpeter) [inline perun-cli/anonymizeUser:14]: Please align the indentation with the other options.

**Review by balcirakpeter — CHANGES_REQUESTED** [review]

File: perun-cli/anonymizeUser (hunk 1 above)
Comment (Johaney-s) [reply perun-cli/anonymizeUser]: fixed

File: perun-cli/anonymizeUser (hunk 2 above)
Comment (Johaney-s) [reply perun-cli/anonymizeUser]: aligned

**Review by balcirakpeter — APPROVED** [review]

**Pull request merged**